models/patient-x-attribute.js

```diff
@@ -0,0 +1,17 @@
+const   Sequelize   = require("sequelize"),
+        sequelize   = require("../db/sequelize"),
+        User        = require("./user")
+        Attribute   = require("./attribute");
+
+
+let patient_x_attribute = module.exports = sequelize.define('patient_x_attribute', {
```

Comment:
I don't think we need to explicitly create this table using Sequelize. I think we can specify the relationship using a belongsToMany relation and it will create the table for us. An example of this is seen at http://docs.sequelizejs.com/manual/tutorial/associations.html in their n:m relation section with a specific example of tagging things, which is similar to what we are doing here.

Reply:
The creation of the many to many tables were done explicitly to make it clear that those tables existed.  I didn't want devs to have to go looking into the user table to find the logic for the user_x_attribute table for example.  However, I didn't look too deeply into the API's that sequelize offers for tables that are created by the belongsToMany relation.  If you believe that they would be useful, I am all for implementing these tables using the belongsToMany relation.